app/assets/stylesheets/application.scss

```diff
@@ -0,0 +1,9 @@
+// Import locally defined variables. Load this before 'bootstrap'
+@import "variables/*";
+@import "utils/*";
+@import "bootstrap";
+@import "font-awesome-sprockets";
+@import "font-awesome";
+@import "variables/*";
+@import "utils/*";
```

Comment:
Thinking this might be a copy/paste issue. You've imported `utils` twice

Reply:
Yep. Thanks.